workshops/fabric-e2e-rag/workshop.md

```diff
@@ -29,18 +29,18 @@ wt_id: data-114676-jndemenge
 
 In this workshop, we'll demonstrate how to develop a context-aware question answering framework for any form of a document using [OpenAI models](https://azure.microsoft.com/products/ai-services/openai-service), [SynapseML](https://microsoft.github.io/SynapseML/) and [Azure AI Services](https://azure.microsoft.com/products/cognitive-services/). The source of data for this workshop is a PDF document, however, the same framework can be easily extended to other document formats too.
 
-## You'll learn how to: 
+## Goals
 
-- Pre-Process PDF Documents using [Azure AI Document Intelligence](https://azure.microsoft.com/products/ai-services/ai-document-intelligence) in Azure AI Services.
+You'll learn how to:
+
+- Pre-process PDF Documents using [Azure AI Document Intelligence](https://azure.microsoft.com/products/ai-services/ai-document-intelligence) in Azure AI Services.
 - Perform text chunking using SynapseML.
 - Generate embeddings for the chunks using SynapseML and  [Azure OpenAI Services](https://azure.microsoft.com/products/cognitive-services/openai-service).
-- Store the embeddings in a vector store using [Azure AI Search](https://azure.microsoft.com/products/search).
+- Store the embeddings in [Azure AI Search](https://azure.microsoft.com/products/search).
```

Comment:
Azure AI search does not want to be thought of as just a vector store, so there's a lot of rewords like that. (It only added vector storage support recently, actually.) Preferred term is "search engine".